antivir/scanners/scanner_clamav.py

```diff
@@ -20,26 +20,24 @@
 #
 ##############################################################################
 
-from openerp import SUPERUSER_ID
-from openerp.osv import osv, fields
+from odoo import models, fields, api, _, SUPERUSER_ID
 from .pyclamd import ClamdAgnostic
 
 
-class antivir_scanner_clamav(osv.osv):
+class AntivirScannerClamav(models.Model):
     _inherit = 'antivir.scanner'
     _av_engine = 'antivir.scanner.clamav'
 
     def __init__(self, registry, cr):
-        super(antivir_scanner_clamav, self).__init__(registry, cr)
+        super(AntivirScannerClamav, self).__init__(registry, cr)
```

Comment:
I don't know if Odoo is still only python 2.7 ? If not You could use super().__init__(registry, cr)

Reply:
Yes, Odoo 10.0 still yet using python 2.7.